### PR TITLE
Fix

### PR DESCRIPTION
**Pull request by RisingLeaf**
**Bugfix:** This PR addresses issue #7349

## Fix Details
Fix for "The Defector Mission to get the escorts attacking the Syndicate ships

### CODE REVIEW

File: data/governments.txt [hunk 1]
```diff
@@ -730,6 +730,16 @@ government "Syndicate"
 	"hostile hail" "hostile syndicate"
 	raid "pirate raid"
 
+government "Syndicate (Free World Haters)"
+	swizzle 4
+	color 0 .41 .71
+	"display name" "Syndicate"
+	"player reputation" -1000
+	"attitude toward"
+		"Merchant" .3
```
Comment (Amazinite) [inline data/governments.txt:739]: But they don't hate the Free Worlds...

**Comment by Amazinite** [comment]
The reason this bug is happening is because the event "fw at war with Syndicate" is triggered on enter of the system. That event properly sets the Syndicate as hostile to the Free Worlds, but the effects of events are currently always delayed by one day. This wasn't a problem before because the event was triggered sooner, and a change from #7000 is what moved the event to where it is.

I plan to redo this mission in the future, as the changes to this mission from #7000 were just slight tweaks as a hold over until I can spend more time on it. As such, I'd prefer if this government were defined above this mission instead of in the government file, since it's temporary.

**Comment by RisingLeaf** [comment]
done

File: data/human/free worlds 2 middle.txt [hunk 2]
```diff
@@ -2018,7 +2018,17 @@ event "end battle of alioth"
 		fleet "Large Republic" 4000
 		fleet "Navy Surveillance" 1600
 
-
+government "Syndicate (Hostile)"
```
Comment (Amazinite) [inline data/human/free worlds 2 middle.txt:2021]:
```suggestion


government "Syndicate (Hostile)"
```

**Review by Amazinite — APPROVED** [review]

**Pull request merged**